src/execution/executor.md

```diff
@@ -16,9 +16,9 @@ completed.
 In this section, we'll write our own simple executor capable of running a large
 number of top-level futures to completion concurrently.
 
-For this one, we're going to have to include the `futures` crate in order to
-get the `FutureObj` type, which is a dynamically-dispatched `Future`, similar
-to `Box<dyn Future<Output = T>>`. `Cargo.toml` should look something like this:
+For this one, to get the `FutureObj` type, a dynamically-dispatched `Future`,
```

Comment:
IMO this newer version is harder to read-- I view it as explaining why we bring in `futures`, rather than explaining how to get `FutureObj`, a thing we didn't even know we needed. Also, this is terribly out of date since FutureObj isn't even required anymore.

Reply:
Interesting, this is not what I read the original version as.